### ISSUE TITLE
Error uploading map list, please try again (Upload to Pastebin)

### ISSUE THREAD

**Issue by wakasm** [briansd9/exile-diary#33]
Any time I try to upload the map list to pastebin, I get an error.  

[2019-09-30 13:16:19] Error uploading map list, please try again

There are no configuations I can find for pastebin, so not 100% sure how to resolve this.

**Comment by briansd9** [comment]
Can you upload me your log file `%appdata%\exile-diary\log.txt`? 

There should be a line "Error uploading to pastebin" followed by the error message.

**Comment by wakasm** [comment]
[log.txt](https://github.com/briansd9/exile-diary/files/3679446/log.txt)

Here you go.  It started working again today.  But the error is in the log.

**Comment by wakasm** [comment]
As a random aside, I also noticed that if you go afk inside the azurite mine, it counts towards your total time.  (I guess I afked for like 20 hours somehow in one).  Is there any method to edit this time down?

**Comment by briansd9** [comment]
I see, it was a rate limit... will change to a more appropriate error message.

Currently there's no way to edit from within the app itself, the database file has to be modified directly. Let me know if you'd like more detailed instructions for this

**Comment by wakasm** [comment]
Absolutely.  Your tool is basically the tool I've dreamed of (up until now, I have manually tracked my time played and large item drops in mapped) so learning how to fix the db would also be great if it's not too much of a hassle.

**Comment by briansd9** [comment]
1) Get an SQLite database editor, I use https://sqlitebrowser.org/
2) Open your character's database file `%appdata%\exile-diary\CHARNAME.db` (please make a backup of this first)
4) Go to the `mapruns` table and find the row you want to edit
5) Change the `firstevent` and `lastevent` values as you wish (the "time spent in map" stat is derived from the difference between these two values)
